orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationClient.java

```diff
@@ -40,6 +41,8 @@ public class OrchestrationClient {
             .visibility(PropertyAccessor.GETTER, JsonAutoDetect.Visibility.NONE)
             .visibility(PropertyAccessor.SETTER, JsonAutoDetect.Visibility.NONE)
             .serializationInclusion(JsonInclude.Include.NON_NULL)
+            .deserializerByType(
+                LLMModuleResult.class, new GeneralizedFallbackDeserializer<>(LLMModuleResult.class))
```

Comment:
(Major)

Please decide on which Deserializer to use for `LLMModuleResult`. Your PR ships two classes, one unused. Let's avoid "dead" code.